packages/block-editor/src/store/actions.js

```diff
@@ -987,3 +987,21 @@ export function* insertAfterBlock( clientId ) {
 	);
 	yield insertDefaultBlock( {}, rootClientId, firstSelectedIndex + 1 );
 }
+
+/**
+ * Generator that toggles the highlighted block state.
```

Comment:
This function isn't a generator, it's just a plain action creator.

More generally, I wonder if we shouldn't just standardize on some phrasing like "action dispatcher" or "action creator", considering:

- For developers, there's a high likelihood of copying documentation as prior art from other functions in this file, and we can optimize for this by standardizing on a consistent way to express them.
- For public documentation, there's really no need to differentiate between generators and action creators. Even if strictly speaking this thing we're exporting is either an action creator (plain function) or generator, for the purposes of documentation the only thing the reader should really care about is that this thing causes action(s) to be dispatched.